client/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please consider removing the example code of cypress, if you don't need it anymore.